src/scripts/tab.js

```diff
@@ -0,0 +1,37 @@
+console.log('test');
+
+var tabContent = function () {
```

Comment:
Jeżeli zaczęłaś używać `let` do deklaracji zmiennych to używaj `let` i ` const` zamiast `var` w całym kodzie